### PR TITLE
x11-drivers/nvidia-{drivers, metadata}: Bump to 515.65.01

### PR DESCRIPTION
**Pull request by sayanchowdhury**
# x11-drivers/nvidia-{drivers, metadata}: Bump to 515.65.01

## Testing done

~Jenkins CI running http://jenkins.infra.kinvolk.io:8080/job/container/job/sdk/248/cldsv/~
Jenkins CI passed http://jenkins.infra.kinvolk.io:8080/job/container/job/packages_all_arches/599/cldsv/

- [x] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)
- [ ] Inspected CI output for image differences: `/boot` and `/usr` size, packages, list files for any missing binaries, kernel modules, config files, kernel modules, etc.

### CODE REVIEW

**Comment by sayanchowdhury** [comment]
- The CVE fixes were pushed to 515.65 branch so updating it to the more recent version
- NVD does not seem to have updated the CVE links, so left it empty. Should we link to some other database? /cc @dongsupark

File: changelog/security/2022-09-16-nvidia-drivers.md [hunk 1]
```diff
@@ -0,0 +1 @@
+- nvidia-drivers (CVE-2022-31607, CVE-2022-31608, CVE-2022-31615)
```
Comment (krnowak) [inline changelog/security/2022-09-16-nvidia-drivers.md:1]: Link them to the NVD database, even if those are empty. Eventually they will show up.

**Comment by pothos** [comment]
The files are still installed to the OEM partition. Could we install this package in the base OS instead, for users to get updates? We could add a migration action in update-engine to delete the OEM files if both the new and old `/usr` have the units.

**Comment by pothos** [comment]
To install it on `/usr` it would need to be a dependency of `coreos-base/coreos/coreos-0.0.1.ebuild` (moved from `coreos-devel/board-packages/board-packages-0.0.1.ebuild` and `coreos-base/oem-ec2-compat/oem-ec2-compat-0.1.2-r2.ebuild`/`coreos-base/oem-azure/oem-azure-2.6.0.2-r1.ebuild`)

**Comment by sayanchowdhury** [comment]
Closing this PR in favor of https://github.com/flatcar/coreos-overlay/pull/2480

**Pull request closed without merging**